### DISCUSSION TITLE
How to report issues with cloud image issues?

### DISCUSSION THREAD

**Discussion by alexpdp7** [cockpit-project/cockpit-machines#1532] [General]
Hi!

I can create a Ubuntu 22.04 VM through the cloud image, and user configuration is done correctly, but the hostname is not set properly. I am not sure where in the chain the problem lies- I'll start looking into it, but in the meantime I'm wondering if there's a good place to discuss these issues.

edit: the Almalinux 9 does not work either, testing with [a RHEL image](https://access.redhat.com/downloads/content/479/ver=/rhel---9/9.3/x86_64/product-software) now... edit2: and yeah, that gets `localhost` has the hostname too)

Cheers,

Álex

**Comment by jelly** [comment]
Do you expect the hostname to be the virtual machine name?

**Reply by alexpdp7** [reply]
Well, maybe, or maybe provide an option to set the hostname through cloud-init- it doesn't have be necessarily the VM name.
